src/comments.js

```diff
@@ -4,10 +4,12 @@ class Comment {
     this.parentId = parentId;
     this.text = text;
   }
+
```

Comment:
Nice work! Very great utilization of all of our knowledge that we have acquired thus far. I am just floored by our progress as a group. I mean, you folks are just swell.